validphys2/src/validphys/closuretest/multiclosure_pseudodata.py

```diff
@@ -27,14 +28,8 @@ def fits_dataset_cvs(fits_dataset):
     """
     fits_cv = []
     for ds in fits_dataset:
-        # using the official loader is really slow, open the CSV
-        # and then cut the central values manually.
-        # TODO: Save central values in nice table like pseudodata
-        # but this should be done beyond NNPDF4.0
-        cd_df = pd.read_csv(ds.commondata.datafile, sep=r'\s+', skiprows=1, header=None)
-        # based on columns from python cd reader:
-        # ['entry', 'process', 'kin1', 'kin2', 'kin3', 'data', 'stat']
-        fits_cv.append(cd_df.iloc[cut_mask(ds.cuts), 5].to_numpy())
+        cd_df = ds.commondata.metadata.load_data_central()
+        fits_cv.append(cd_df.iloc[cut_mask(ds.cuts)].to_numpy())
```

Comment:
What do you mean?
The central data should be different per fit in here in level_1 fits.

Reply:
This function is basically only used by some functions that I added (about this point I am currently checking if it used somewhere else). At the moment it is made to work with a list of fits but I am currently using it for a single fit which means that the for loop is useless. I am just wondering if we could make it work with a single fit and maybe define a collect if one needs a list of fits.

Reply:
For a single fit you are already getting the right values.

As far as I understand it this is because 

```
# NOTE: for some reason the fit doesn't get properly resolved if you try to
# collect data over fits
```

Reply:
But in any case, this should be taken elsewhere. This should be only for the runcard :___

For your PR, feel free to remove functions that are not being used anywhere else. The fewer functions the better.

Reply:
Indeed, but maybe it is time to understand the "for some reason"